blog/20140317-galculator-coming-to-mate.de.md

```diff
@@ -9,7 +9,7 @@
 -->
 
 Das MATE-Team verkündet stolz die Zusammenarbeit mit dem 
-[galculator](http://galculator.sourceforge.net/)-Team, und dass galculator
+[galculator](https://github.com/galculator/galculator)-Team, und dass galculator
```

Comment:
same here

(offtopic, seems to be dead project galculator, no more updates since 2015)